tests/test_mode_manager_llm.py

```
@@ -0,0 +1,59 @@
+    file_path = tmp_path / "llm_test.py"
+    file_path.write_text("print('hello')\n")
+    orig = file_path.read_text()
+    # Simulate LLM edit
+    new_content = orig + "# LLM EDIT\n"
+    change = FileChange(str(file_path), orig, new_content, "fix")
+    mm.add_pending_change(change)
+    # Patch can_edit_file to always allow
+    mm.permissions[str(file_path)] = PermissionLevel.ALL
+    mm.apply_change(change)
+    # Check audit log
+    log_path = os.path.join(os.path.dirname(str(file_path)), "../sessions/edit_audit.log")
```

Ruff (0.12.2)

Local variable `log_path` is assigned to but never used

Remove assignment to unused variable `log_path`

(F841)